op-service/ioutil/streams.go

```diff
@@ -0,0 +1,52 @@
+package ioutil
+
+import (
+	"io"
+	"os"
+)
+
+var (
+	stdOutStream OutputTarget = func() (io.Writer, io.Closer, Aborter, error) {
+		return os.Stdout, &noopCloser{}, func() {}, nil
+	}
+)
+
+type Aborter func()
+
+type OutputTarget func() (io.Writer, io.Closer, Aborter, error)
+
+func NoOutputStream() OutputTarget {
+	return func() (io.Writer, io.Closer, Aborter, error) {
+		return nil, nil, nil, nil
+	}
+}
+
+func ToAtomicFile(path string, perm os.FileMode) OutputTarget {
+	return func() (io.Writer, io.Closer, Aborter, error) {
+		f, err := NewAtomicWriterCompressed(path, perm)
+		if err != nil {
+			return nil, nil, nil, err
+		}
+		return f, f, func() { _ = f.Abort() }, nil
+	}
+}
+
+func ToStdOut() OutputTarget {
+	return stdOutStream
+}
+
+func ToStdOutOrFile(outputPath string, perm os.FileMode) OutputTarget {
```

Comment:
(optional) naming nit:
```suggestion
func ToStdOutOrFileOrNoop(outputPath string, perm os.FileMode) OutputTarget {
```